ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/hardware_counters.yml

```diff
@@ -0,0 +1,14 @@
+hardware_counters:
+  features:
+    - vlan-interface: in
+    # in 3.8, this will trigger a warning in eos_cli_config_gen because of the
```

Comment:
This comment is not true since you are also loosening the schema

Reply:
Addressed in [2da3646](https://github.com/aristanetworks/ansible-avd/pull/2447/commits/2da364635e26f1c98e6487c965da30cf48cc2b0f)